integreat_cms/cms/templates/pois/poi_list.html

```diff
@@ -60,7 +61,7 @@ <h1 class="heading">{% trans 'Locations on map' %}</h1>
         <tbody>
         {% for poi in pois %}
             {% get_translation poi language.slug as poi_translation %}
-            {% include "pois/poi_list_row.html" %}
+            {% include "pois/poi_list_row.html" with LANGUAGE_CODE=LANGUAGE_CODE backend_language=backend_language %}
```

Comment:
```suggestion
            {% include "pois/poi_list_row.html" %}
```
Is this really necessary? I thought the context is passed to the included template anyway, the `with` parameters are only required, if the name differs...